cpp/include/cudf/strings/detail/utilities.hpp

```diff
@@ -72,7 +72,7 @@ rmm::device_vector<string_view> create_string_vector_from_column(
  * @return Child offsets column
  */
 std::unique_ptr<cudf::column> child_offsets_from_string_vector(
-  const rmm::device_vector<string_view>& strings,
+  rmm::device_uvector<string_view> const& strings,
```

Comment:
Since you're already changing the parameter, we should avoid using owning types as function parameter and prefer things like `host/device_span`.